src/components/Timeline/updates/TimelineJourneyInstance.tsx

```diff
@@ -0,0 +1,105 @@
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Box, Button, Fade, Typography } from '@material-ui/core';
+import { FormattedMessage, useIntl } from 'react-intl';
+import React, { useEffect, useRef, useState } from 'react';
+
+import UpdateContainer from './elements/UpdateContainer';
+import ZetkinPersonLink from 'components/ZetkinPersonLink';
+import { ZetkinUpdateJourneyInstance } from 'types/updates';
+
+interface Props {
+  update: ZetkinUpdateJourneyInstance;
+}
+
+const TimelineJourneyInstance: React.FunctionComponent<Props> = ({
```

Comment:
At some point (probably post release) I think we should go over these components and find a consistent way of naming them based on the underlying events that they render. It wasn't obvious to me a first what event this was for.

Reply:
Todo todo